crawl-ref/source/dat/des/branches/lair.des

```diff
@@ -476,7 +476,7 @@ FTILE:   Ll~ = floor_dirt
 SUBST:   T : TG
 KFEAT:   # = iron_grate
 KFEAT:   O = enter_lair
-MONS:    gnoll ; whip
+KMONS:   z = gnoll name:zookeeper n_rpl n_des n_noc ; whip
```

Comment:
The zookeeper can probably be a real monster rather than popcorn, since you aren't making the player fight anything else. Any midlevel humanoid with a whip is probably fine, maybe an orc knight? I'm not sure the rename is really necessary, either.